README.md

```
@@ -1,238 +1,92 @@
-<a href="https://www.youtube.com/watch?v=tfVYXpjyGqQ" target="_blank">
-  <img src="assets/cluster-api-yt.png" alt="Cluster API YT">
-</a>
 
-### Satellite
-Satellites are the core building blocks for exosphere. They are lego blocks designed for a specific purpose: you can connect them together to create complex systems in a matter of minutes without worrying about the underlying infrastructure.
 
-They are pre-implemented serverless functions highly optimized for workflows and high volume batch processing, optimized for cost, reliability, developer velocity and ease of use.
+## Getting Started
 
-Our in-house optimization for workflows and batch processing can lead to significant cost savings, for example you can expect a cost per token saving of about 50-75% on LLMs like DeepSeek R1 70B, Gemma 3 27B, etc.
+- ### Installation
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)